types/index.d.ts

```diff
@@ -169,7 +169,7 @@ declare module 'highlight.js' {
         disableAutodetect?: boolean
         contains: (Mode)[]
         case_insensitive?: boolean
-        keywords?: Record<string, any> | string
+        keywords?: string | string[] | Record<string, string | string[]>
```

Comment:
We typically use `Array<type>` for arrays not `[]`.  Can we do that here also for consistency?

Reply:
In this file I can see one `Array<` and 24 instances of `[]`.
That only instance is `Array<string | RegExp>` so it must use this syntax.
I suggest keeping `[]` where they are possible in this file.